libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -75,12 +75,12 @@
         AztecToolbarClickListener,
         HistoryListener {
 
-    private static final String ARG_PARAM_TITLE = "param_title";
-    private static final String ARG_PARAM_CONTENT = "param_content";
-
-    private static final String KEY_TITLE = "title";
-    private static final String KEY_CONTENT = "content";
-
+    private static final String ATTR_ALIGN_ = "align-";
```

Comment:
I think this convention of using a `_` at the end of the variable name could lead to errors. While reviewing this, I found it very easy to confuse `ATTR_SIZE` with `ATTR_SIZE_` because they look so similar. Perhaps there's a better suffix we can use, like `_DASH` (so `ATTR_SIZE_DASH`)?